docs/content/reference/api.md

```diff
@@ -3856,6 +3858,9 @@ AWSPlatformSpec
 <tbody><tr><td><p>&#34;AWS&#34;</p></td>
 <td><p>AWSPlatform represents Amazon Web Services infrastructure.</p>
 </td>
+</tr><tr><td><p>&#34;Agent&#34;</p></td>
+<td><p>AgentPlatform represents user supplied insfrastructure booted with agents.</p>
+</td>
```

Comment:
We probably need some docs somewhere that describe the management cluster dependencies that allow the agent provider to work (I'm not sure exactly where, but without prior knowledge this doc isn't self-explanatory like the cloud provider platforms)

Reply:
You're right, I opened a [task](https://issues.redhat.com/browse/MGMT-8469) on that